pkg/controller/kogitobuild/build/build_config_test.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2020 Red Hat, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package build
+
+import (
+	"github.com/kiegroup/kogito-cloud-operator/pkg/apis/app/v1alpha1"
+	buildv1 "github.com/openshift/api/build/v1"
+	"github.com/stretchr/testify/assert"
+	v12 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"testing"
+)
+
+func Test_decoratorForSourceBuilder_setIncrementalBuild_Test(t *testing.T) {
```

Comment:
Can you please also add test checking incremental build being disabled in case `DisableIncremental` is set to true?

Reply:
Added test